llvm/lib/TargetParser/Triple.cpp

```diff
@@ -916,6 +921,7 @@ static Triple::ObjectFormatType getDefaultFormat(const Triple &T) {
   case Triple::renderscript64:
   case Triple::riscv32:
   case Triple::riscv64:
+  case Triple::rl78:
```

Comment:
I don't believe this is covered in the unit test

Reply:
Done. Thank you!